ext/ForwardDiffStaticArraysExt.jl

```diff
@@ -10,6 +10,8 @@ using ForwardDiff: Dual, partials, npartials, Partials, GradientConfig, Jacobian
                    vector_mode_jacobian, vector_mode_jacobian!, valtype, value
 using DiffResults: DiffResult, ImmutableDiffResult, MutableDiffResult
 
+ForwardDiff.supports_fast_scalar_indexing(::StaticArray) = true
```

Comment:
Is this actually true? `StaticArray` is the abstract supertype, is it required that all subtypes support fast scalar indexing?